src/app/code/community/Shopgate/Cloudapi/Model/Frontend/Observer/Layout.php

```diff
@@ -31,9 +41,32 @@ public function execute(Varien_Event_Observer $observer)
             return $this;
         }
 
-        $layout = $observer->getEvent()->getLayout()->getUpdate();
-        $layout->addHandle('shopgate_cloudapi_default');
+        /** @var Mage_Core_Model_Layout $layout */
+        $layout = $observer->getEvent()->getLayout();
+        $layout->getUpdate()->addHandle('shopgate_cloudapi_default');
+        $this->addCustomHandle($layout);
 
         return $this;
     }
+
+    /**
+     * @param Mage_Core_Model_Layout $layout
+     */
+    private function addCustomHandle($layout)
+    {
+        /** @var Mage_Core_Controller_Request_Http $request */
+        $request = Mage::app()->getRequest();
+        $path    = sprintf(
+            "/%s/%s/%s",
```

Comment:
should be single quotes, I am fairly certain the %s should still work

Reply:
changed